cloudsql/cloudsql_deployment.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: wordpress
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: wordpress
+  template:
+    metadata:
+      labels:
+        name: wordpress
+    spec:
+      containers: 
+        - image: wordpress
+          name: web
+          env:
+            - name: WORDPRESS_DB_HOST
+              # Connect to the SQL proxy over the local network on a fixed port.
+              value: 127.0.0.1:3306
+            # These secrets are required to start the pod. 
+            - name: WORDPRESS_DB_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: cloudsql
+                  key: password
+            - name: WORDPRESS_DB_USER
+              valueFrom:
+                secretKeyRef:
+                  name: cloudsql
+                  key: username
+          ports:
+            - containerPort: 80
+              name: wordpress
+        - image: b.gcr.io/cloudsql-docker/gce-proxy
```

Comment:
They don't appear to have any tags pushed other than latest, so I used a digest.